lib/core/src/client/preview/start.js

```diff
@@ -420,11 +420,12 @@ export default function start(render, { decorateStory } = {}) {
             `${kindName} => ${name || key}: story.parameters.decorators is deprecated; use story.decorators instead.`)();
           }
           const decoratorParams = decorators ? { decorators } : null;
-          const displayNameParams = name ? { displayName: name } : {};
-          kind.add(storyNameFromExport(key), storyFn, {
+          const exportName = storyNameFromExport(key);
+          const idParams = { __id: toId(kindName, exportName) };
```

Comment:
Should we only pass this if `name` is defined?

Reply:
Hmm could see arguments both ways. I think it's simpler to do the calculation here so it's all in one place.